app/src/main/res/values/strings.xml

```diff
@@ -416,6 +416,6 @@
     description="The default hint text for the ratio interaction if the placeholder text is not available.">
     Enter a ratio in the form x:y.
   </string>
-
-
+  <string name="smallest_textsize">Smallest textSize</string>
```

Comment:
Suggested change:
`<string name="smallest_text_size">Smallest text size</string>`

Reply:
@Mehul-Bisht You have missed this suggestion